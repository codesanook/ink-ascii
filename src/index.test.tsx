import React from "react";
import Ascii from "./index";
import { render } from "ink-testing-library";

it("renders correctly", () => {
  const { lastFrame } = render(
    <Ascii font="Slant Relief" horizontalLayout="default" verticalLayout="default" text="Yeah!" />
  );
  expect(lastFrame()).toMatchSnapshot();
});
