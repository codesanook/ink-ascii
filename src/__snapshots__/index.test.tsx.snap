// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"__/\\\\\\\\\\\\________/\\\\\\\\\\\\________________________________/\\\\\\\\\\\\_____________/\\\\\\\\\\\\____
 _\\\\///\\\\\\\\\\\\____/\\\\\\\\\\\\/________________________________\\\\/\\\\\\\\\\\\___________/\\\\\\\\\\\\\\\\\\\\\\\\\\\\__
  ___\\\\///\\\\\\\\\\\\/\\\\\\\\\\\\/__________________________________\\\\/\\\\\\\\\\\\__________/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\_
   _____\\\\///\\\\\\\\\\\\/__________/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\___/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\____\\\\/\\\\\\\\\\\\_________\\\\//\\\\\\\\\\\\\\\\\\\\\\\\\\\\__
    _______\\\\/\\\\\\\\\\\\_________/\\\\\\\\\\\\/////\\\\\\\\\\\\_\\\\////////\\\\\\\\\\\\___\\\\/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\___\\\\//\\\\\\\\\\\\\\\\\\\\___
     _______\\\\/\\\\\\\\\\\\________/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\____/\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\__\\\\/\\\\\\\\\\\\/////\\\\\\\\\\\\___\\\\//\\\\\\\\\\\\____
      _______\\\\/\\\\\\\\\\\\_______\\\\//\\\\\\\\///////____/\\\\\\\\\\\\/////\\\\\\\\\\\\__\\\\/\\\\\\\\\\\\___\\\\/\\\\\\\\\\\\____\\\\///_____
       _______\\\\/\\\\\\\\\\\\________\\\\//\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\_\\\\//\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\/\\\\\\\\_\\\\/\\\\\\\\\\\\___\\\\/\\\\\\\\\\\\_____/\\\\\\\\\\\\____
        _______\\\\///__________\\\\//////////___\\\\////////\\\\//__\\\\///____\\\\///_____\\\\///_____"
`;
